doc/user/zebra.rst

```diff
@@ -796,6 +812,15 @@ Allocated label chunks table can be dumped using the command
    Proto ospf: [20000/21000]
    Proto isis: [22000/23000]
 
+.. clicmd:: mpls label dynamic-block (16-1048575)
```

Comment:
Incomplete command according to the CLI. start/end should be specified, not start only.

Reply:
done